data/clients.json

```diff
@@ -208,14 +208,14 @@
         "name": "eyeCU",
         "platforms": [
             "Linux",
-            "Other",
+            "OS/2",
             "Windows"
         ],
         "url": "https://eyecu.opiums.eu"
     },
     {
         "doap": null,
-        "last_renewed": null,
+        "last_renewed": "2021-03-03T05:30:05",
```

Comment:
Maybe this is getting a bit emotional, but I don't think we should put efforts in placing Pidgin here. The XMPP implementation is really old and as long as the maintainer comes here, I would refuse to update this.

Reply:
> The XMPP implementation is really old

... and still the best among what Pidgin supports out-of-box at all. The implementations for IRC and Bonjour are way more primitive, and the rest of prpls seem to be just a pile of dead code for dead protocols.

If Pidgin is not an XMPP-first client anymore, then what is that, just a framework for great third-party prpls? ;)

> as the maintainer comes here

Should I notify them?

Reply:
@bodqhrohro - I recommend to do the following: 

Reach out the developer of interest and ask to update the timestamp **IF**

- they are still actively developing and maintaining their project / XMPP implementation of their project
- think it is worth to guide new people to their project

Reply:
For all the stale projects we might create a separate archive collection